test/controller_SUITE.erl

```diff
@@ -104,9 +104,10 @@ remove_users_scenario_not_started_fails(_) ->
 
 check_status_with_running_users_is_correct(_) ->
     do_start_scenario(testing_scenario, regular_vars()),
-    amoc_controller:add_users(1, 10),
-    Status = amoc_controller:get_status(),
-    ?assertMatch({running, testing_scenario, _, 10}, Status).
+    StartId = 6,
+    EndId = 10,
+    amoc_controller:add_users(6, 10),
```

Comment:
why not `amoc_controller:add_users(StartId, EndId),`
anyway, that's not blocking.